app/src/main/res/layout-sw600dp-land/concept_card_fragment.xml

```diff
@@ -0,0 +1,84 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.topic.conceptcard.ConceptCardViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorConceptCardBackground">
+
+    <androidx.appcompat.widget.Toolbar
+      android:id="@+id/concept_card_toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="@color/colorConceptToolbarBackground"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      app:title="@string/concept_card_toolbar_title"
+      app:titleTextAppearance="@style/ToolbarTextAppearance"
+      app:titleTextColor="@color/white" />
+
+    <FrameLayout
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/concept_card_toolbar">
+
+      <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:overScrollMode="never"
+        android:paddingBottom="112dp"
```

Comment:
I think this should be `138` as per mocks.

Reply:
We had a discussion on the paddings in scrollable screens and we decided that the margin (or the padding) between the bottommost item and the bottom of the screen should be 160 including the android bar whose height is 48.  
So, we should use 112 in the code (160 - 48).  
@rt4914 PTAL